corehq/apps/export/templates/export/partials/table.html

```diff
@@ -188,10 +188,18 @@ <h4 data-bind="css: {'text-muted': hasEmailedExport && !isAutoRebuildEnabled()}"
                                     button and download the file.
                                 {% endblocktrans %}
                             </p>
-                            <p data-bind="if: emailedExport.taskStatus && emailedExport.taskStatus.justFinished()">
+
+                            <p class="text-success" data-bind="if: emailedExport.taskStatus && emailedExport.taskStatus.justFinished() && emailedExport.taskStatus.success()">
                                 <i class="fa fa-check"></i>
                                 <strong>{% trans "Data update complete" %}</strong>
                             </p>
+
+                            <p class="text-danger" data-bind="if: emailedExport.taskStatus && emailedExport.taskStatus.justFinished() && emailedExport.taskStatus.failed()">
+                                <i class="fa fa-exclamation-triangle"></i>
+                                <strong>{% trans "Data update failed!" %}</strong>
+                                {% blocktrans %}If this problem persists, please <a href="#modalReportIssue" data-toggle="modal">Report an Issue</a>.{% endblocktrans %}
```

Comment:
Is there anything they should do? Should they click the Update Data button?

Reply:
dunno that I'd recommend it - they'll probably spam that button anyways, but odds are it's a bug that'll fail every time